templates/mysql.yml

```diff
@@ -1,8 +1,11 @@
 test:
     adapter: mysql2
-    encoding: utf8
+    encoding: utf8mb4
```

Comment:
How about make `ENV['WERCKER_MYSQL_ENCORDING']`, set default is `utf8` and use it?